spec/fixtures/single-3way-diff-complex.txt

```diff
@@ -0,0 +1,23 @@
+<<<<<<< HEAD
+These are my changes
+||||||| merged common ancestors
+<<<<<<< Temporary merge branch 1
+    Refer: http://stackoverflow.com/questions/16990657/git-merge-diff3-style-need-explanation
```

Comment:
Wow, yeah, that can get kind of crazy. I wonder if it can nest arbitrarily with multiple criss-cross merges... ? In that case we'd probably need some kind of recursive descent parser just to identify marked conflicts in the first place.